src/app/ubs-user/ubs-user-messages/notification-body/notification-body.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
+import { UserMessagesService } from '../../services/user-messages.service';
+
+@Component({
+  selector: 'app-notification-body',
+  templateUrl: './notification-body.component.html',
+  styleUrls: ['./notification-body.component.scss'],
+  encapsulation: ViewEncapsulation.ShadowDom
+})
+export class NotificationBodyComponent implements OnInit {
+  @Input() body: string;
+
+  constructor(public userMessagesService: UserMessagesService) {}
+
+  ngOnInit(): void {}
```

Comment:
remove hook and implements cuz it is empty

Reply:
Done